index.html

```diff
@@ -248,43 +248,66 @@ <h2>Betekenisvol verbinden van informatie met BP4mc2-praktijkervaringen</h2>
     <p>Omdat de identiteit van een subject in linked data heel specifiek is moeten we in onze modellering van de klassen waartoe deze subjects kunnen behoren evenzo specifiek zijn.<br>
+      <!-- even uitleggen wat normalisatie hier betekent. Hoe 'normaler', hoe dichter bij de werkelijkheid. Hoe 'denormaler', hoe dichter bij de uiteindelijke implementatievorm. Een plaatje zou hierbij handig zijn. Idealiter vinden we een andere tekst die dit uitlegt.-->
       Voordat je begint met het transformeren van een informatiemodel naar een linked data model is het belangrijk om stil te staan bij het oorspronkelijke doel en de uitgedrukte betekenis van het informatiemodel. Is het een logisch datamodel? Een technisch objectmodel? Een technisch berichtmodel?<br>
       De meeste informatiemodellen zijn in meer of mindere mate gedenormaliseerd, waarbij delen tot aan de tweede normaalvorm. Dit is omdat de informatiemodellen vanuit een bepaald oogpunt zijn opgesteld en vaak een technisch objectgeoriënteerde insteek hebben. Bijvoorbeeld een berichtmodel voor geoptimaliseerde communicatie, waarbij selectief aangebrachte redundantie voordelig kan zijn.</p>
     <p>Een goed linked data model, aan de andere kant, is een model dat een representatie biedt die zo dicht mogelijk bij de te beschrijven werkelijkheid ligt. Ofwel, een goed linked data model is een genormaliseerd model. De normaalvorm uit de relationele theorie die hier het dichtste bijkomt is de zesde normaalvorm.</p>
+    <!-- behouden, iets duidelijker opschrijven-->
     <p>Normalisatie is dus uiterst belangrijk voor een goed linked data model. Maar, normalisatie is alleen mogelijk wanneer je de betekenis van de data - de semantiek - begrijpt. Hieruit volgt dat het meestal niet mogelijk is om een automatische vertaling te doen van een informatiemodel naar een goed linked data model.</p>
     <p>Merk op dat het wel mogelijk is om een gedenormaliseerd model over te zetten in een correct (gedenormaliseerd) linked data model. Echter, gaat dit in tegen de principes van linked open data. De gedenormaliseerde entiteitsrepresentatie is namelijk per definitie niet meer herkenbaar in de open wereld. Het is verworden tot een beschrijving van iets wat enkel waarde heeft binnen een zeer specifieke context, daarmee de herbruikbaarheid en de linkbaarheid van data verminderend.</p>
 
     <!-- Sectie 5.2.1 -->
     <section>
       <h3>Essentie van het ding</h3>
+      <!-- wat gingen we hier opschrijven?-->

```

Comment:
Dit heeft raakvlakken met normalisatie. Het idee hier was om aan te geven dat wanneer je een klasse modelleert, dat je de alleen die eigenschappen aan een klasse moet toevoegen die direct tot deze dingen behoren, ofwel essentiele eigenschappen zijn. Dit is misschien meer een guideline die volgt uit dit verhaal dan een aparte paragraaf.

Reply:
Ik heb de twee subsecties "Essentie van het ding" en "Gegevens over gegevens" samengevoegd met de para over normalisatie.